src/main/java/net/mcreator/minecraft/ElementUtil.java

```diff
@@ -150,7 +151,7 @@ public static List<MCItem> loadBlocksAndItems(Workspace workspace) {
 	 */
 	public static List<MCItem> loadBlocks(Workspace workspace) {
 		List<MCItem> elements = new ArrayList<>();
-		workspace.getModElements().stream().filter(element -> element.getType().getBaseType() == BaseType.BLOCK)
+		workspace.getModElements().stream().filter(element -> element.getGeneratableElement() instanceof IMCItemProvider)
```

Comment:
Is this check even needed? Filtering is done in forEach for block type, so this can probably omitted, as it is for example in loadBlocksAndItems